### PR TITLE
Support Plurr preview for simple placeholders #468

### PR DESCRIPTION
**Pull request by benjamimo1**
For example {FOO}

### CODE REVIEW

File: pootle/static/js/editor/utils/strings.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ export const Formats = {
   PLURR: 'PLURR',
 };
 
-/**
- * Returns the string format detected for `value`.
- */
 export function detectFormat(value) {
   if (value.search(/{[^{}]*:.*?}/) !== -1) {
```
Comment (iafan) [inline pootle/static/js/editor/utils/strings.js:15]: I would suggest adding comments that say that the first case matches Plurr choice strings, e.g. `{FOO:xxxxx}`, and the other one matches simple placeholders like `{FOO}`.

File: pootle/static/js/editor/utils/strings.js [hunk 2]
```diff
@@ -11,12 +11,12 @@ export const Formats = {
...
     return Formats.PLURR;
+  } else if (value.search(/{[^ ]*}/) !== -1) {
```
Comment (iafan) [inline pootle/static/js/editor/utils/strings.js:17]: I think this regular expression will also match `{}`. While technically it is allowed in Plurr, let's make sure we test this locally on such a string and make sure it doesn't break the UI.

Comment (benjamimo1) [reply pootle/static/js/editor/utils/strings.js:17]: I'll check

**Comment by benjamimo1** [comment]
{PENDING} git rebase on commits that belong to a past branch

**Comment by benjamimo1** [comment]
This issue will be worked on a new branch (identical to this one) but with a conflicting commit removed. So, will close this PR and create a new one.

File: pootle/static/js/editor/utils/strings.js (hunk 1 above)
Comment (benjamimo1) [reply pootle/static/js/editor/utils/strings.js:15]: will do

File: pootle/static/js/editor/utils/strings.js (hunk 2 above)
Comment (benjamimo1) [reply pootle/static/js/editor/utils/strings.js:17]: image

The UI does not break, but the specific case is not addressed. As discussed a separated issue has been created to work on this.

**Pull request closed without merging**